tests/unit/test_config.py

```diff
@@ -379,8 +379,10 @@ def test_can_load_python_version():
         expected_runtime = 'python3.9'
     elif minor <= 10:
         expected_runtime = 'python3.10'
```

Comment:
Also here, we are Py 3.7+ if not 3.8+, a lot can disappear

Reply:
Yeah, there's a number of places in the tests that still reference older versions of python that can be removed.  I'll send a follow up pull request to clean these up.